stix_shifter_modules/reaqta/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -282,7 +282,7 @@
               "object": "certInfo"
             }
           },
-          "filename": [
+          "fsName": [
```

Comment:
you need to change the object values as well since it is mapped to file.name: `"object": "accessorProcess_file"`